src/LuaInterface.cpp

```diff
@@ -71,17 +71,17 @@ QStringList LuaInterface::varName(TVar* var)
         return names;
     }
     names << var->getName();
-    TVar* p = var->getParent();
-    while (p && p->getName() != "_G") {
-        names.insert(0, p->getName());
-        p = p->getParent();
+    TVar* parent = var->getParent();
+    while (parent && parent->getName() != "_G") {
+        names.insert(0, parent->getName());
+        parent = parent->getParent();
     }
     return names;
 }
 
-bool LuaInterface::validMove(QTreeWidgetItem* p)
+bool LuaInterface::validMove(QTreeWidgetItem* widgetItem)
```

Comment:
Qt uses this convention... for example https://doc.qt.io/qt-5/qwidget.html#setupUi. How about we follow it here?

Reply:
TBH I'm not a great fan of that Qt convention because it hides the fact that the argument is a pointer - whilst it is obvious when explaining a method in a help file, because of the `*` in that place - it doesn't help elsewhere, perhaps when quoting a bit of code during an on-line discussion. Our code has such a long history and had so many editors that getting to a uniform standard throughout the whole code-base is going to be a long and to be perfectly honest, not a particularly rewarding process. Whilst it would be nice to have done we probably need to focus our non-infinity energies on fixing actual code defects and getting in the features we or our users want.

:man_shrugging:

Reply:
That is fair.